### PR TITLE
enhancement(elasticsearch sink): add pipeline config

### PR DESCRIPTION
**Pull request by francescop**
feat(sinks:elasticsearch): add pipeline config

closes issue #2691

### CODE REVIEW

File: website/docs/reference/sinks/elasticsearch.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ endpoint][urls.elasticsearch_bulk].
   host = "http://10.24.32.122:9000" # optional, no default
   id_key = "id" # optional, no default
   index = "vector-%F" # optional, default
+  pipeline = "pipeline-name" # optional, no default
```
Comment (bruceg) [inline website/docs/reference/sinks/elasticsearch.md:77]: Note that this file is automatically generated. This change should be accomplished by adding the option to `.meta/sinks/elasticsearch.toml.erb` and running `make generate`.

**Review by bruceg — CHANGES_REQUESTED** [review]
This looks good, just need to complete the docs.

File: website/docs/reference/sinks/elasticsearch.md (hunk 1 above)
Comment (francescop) [reply website/docs/reference/sinks/elasticsearch.md:78]: pardòn. pr updated

**Review by francescop — COMMENTED** [review]
updated documentation, thanks

**Review by bruceg — APPROVED** [review]

**Review by lukesteensen — APPROVED** [review]

**Comment by lukesteensen** [comment]
Thanks for this @francescop! Could you run `make generate` and push a commit with the changes? This otherwise looks ready to merge.

**Comment by francescop** [comment]
> Thanks for this @francescop! Could you run `make generate` and push a commit with the changes? This otherwise looks ready to merge.

did run that again, it changed only the `vector.spec.toml`.
should be all good now.

thanks

**Pull request merged**